Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7-alpine
+
+WORKDIR /app
+ADD . /app
+
+RUN apk update && apk add linux-headers postgresql-dev gcc python3-dev musl-dev
+
+RUN python3 -m venv env
+RUN source env/bin/activate

```

Comment:
These lines were causing a "no such file or directory" error when I ran `docker compose up` so I just removed them -- I don't think we need to use a virtual environment when this is the only Python app we're going to be running on this container, right?

Reply:
Got the same error. Changing `RUN python3 -m venv env` to `RUN python -m venv env` also seems to work

Reply:
Agreed on removing - these have been removed.